src/components/App/index.js

```diff
@@ -21,14 +21,15 @@ class App extends Component {
     chatOpen: true,
     messages: [],
     textBox: '',
+    network: '',
     theme: MESSENGER, // wrapped with theme provider + HOC
   };
 
   componentDidMount () {
     this.socket = createSocket(this);
   }
 
-  // Event Handlers
+  // --- UI / Event Methods
```

Comment:
it's way more subtle then what it used to be.

At least this way I can differentiate between a `//` comment and a section `//---`